chucks-pick3-web/angular-app/src/app/directives/lowercase-validator.directive.ts

```diff
@@ -0,0 +1,41 @@
+import {NG_VALIDATORS, FormControl, ValidatorFn, Validator} from '@angular/forms';
+import {FormValidationService} from '../services/form-validation.service/form-validation.service';
+import {Directive} from '@angular/core';
+
+// @ts-ignore
+// @ts-ignore
+@Directive({
+  // tslint:disable-next-line:directive-selector
+  selector: '[lowercaseValidator] [ngModel]',
+  providers: [
+    {
+      provide: NG_VALIDATORS,
+      useExisting: LowercaseValidator,
+      multi: true
+    }
+  ]
+})
+export class LowercaseValidator implements Validator {
```

Comment:
Lets make sure we use camel-case here.  Also rename file to `lower-case-validator.directive.ts`.

Reply:
changed file name to lower-case-validator.directive

Reply:
Reverted to pascal case due to conflict.